src/components/Methods.js

```diff
@@ -0,0 +1,95 @@
+import storageManager from './Storage.js';
+import Todolist from './Todolist.js';
+import UpdateUI from './UpdateUI.js';
+
+export default class Methods {
+  constructor(itemsToDelete = [], toogle = false) {
+    this.itemsToDelete = itemsToDelete;
+    this.toogle = toogle;
+    this.Listlength = storageManager.getData().length;
```

Comment:
The name should be in camelCase

Reply:
Thank you for pointing this out, the change has been implemented